ledger-service/http-json/BUILD.bazel

```diff
@@ -192,7 +192,7 @@ daml_compile(
         "IAccount",
         "Disclosure",
     ]],
-    target = lf_version_configuration.get("latest"),  # TODO(SC #13296) remove when interfaces in default
+    target = lf_version_configuration.get("latest"),
```

Comment:
I believe these should be 
```suggestion
    target = lf_version_configuration.get("default"),
```

Reply:
Done everywhere latest was used to ensure interfaces were available.

Reply:
Done everywhere latest was used to ensure interfaces were available.

Reply:
thanks!